Capacitor

planesandunicorns opened the pull request:
Added 3 more paths to the iOS CopyFromResource function to allow copy of pre-populated database from Ionic Capacitor folder structures.
The database resides in this folder structure under Public->dbname or Public->assets->dbname.

planesandunicorns committed:
Update SQLitePlugin.m

planesandunicorns committed:
Update SQLitePlugin.m

Commit:
Added www/assets, public and public/assets to iOS copy paths

Commit:
Added correct NSLog

brody2consult commented:
Thanks @ronwondaal for the contribution. Some more explanation would be helpful. Also should this only apply to iOS?

planesandunicorns commented:
Thats a good question @brodybits, let me check that and if it also applies for Android I’ll put in another pull. 

Ionic/Angular has an assets folder, and all files should be kept in this folder and are written to the www/assets folder on build. Therefor the addition of the www/assets folder. Capacitor no longer builds a ‘www’ folder but instead builds to a ‘public’ folder, so I added ‘public’ and ‘public/assets’ to the pre-populated paths that the plugin is checking.

Commit:
Added new paths for Ionic/Capacitor into

planesandunicorns commented:
@brodybits, I cleaned up the iOS code and added code form android to check multiple paths. Same applied there, Capacitor now writes to public/assets.
It's checked but perhaps review the java code as i'm not much of an android programmer.

HarelM commented:
What's the status of this PR?
I'm looking into migrating my app to capacitor and this seems like something that's probably needed...?
